packages/ng/libraries/user/src/lib/select/input/user-select-input.component.html

```diff
@@ -16,6 +16,12 @@
 		<lu-user-paged-searcher></lu-user-paged-searcher>
 	</header>
 	<lu-user-homonyms></lu-user-homonyms>
+	<lu-option *luUserMeOption="let user" [value]="user">

```

Comment:
thats a design choice that @sanlucca and me took. we'd rather have user-select act the same everywhere even if its not super pertinent, rather than half the user-select who display me as first option, and half who dont